code/modules/mob/mob_movement.dm

```diff
@@ -1,5 +1,5 @@
 /mob/CanPass(atom/movable/mover, turf/target, height=1.5, air_group = 0)
-	if(air_group || (height==0)) return 1
+	if(air_group || (height==0) || (incorporeal_move == INCORPOREAL_ETHEREAL)) return 1

```

Comment:
If this is actually a problem set density = 0 and density = 1 in jaunt, this is spaghetti